build.gradle

```diff
@@ -48,7 +48,7 @@ subprojects {
 }
 
 wrapper {
-    gradleVersion = '5.6.4'
+    gradleVersion = '7.1.1'
```

Comment:
Do we need to bump up the version of gradle too?

Reply:
It has to be bumped to 6.4+, since the version of ml-gradle we've updated to defaults to 7.1.1, I believe 7.1.1 is the best option.